packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -319,9 +326,9 @@ export function usePress(props: PressHookProps): PressResult {
               focusWithoutScrolling(e.currentTarget);
             }
 
-            let stopPressStart = triggerPressStart(e, 'virtual');
-            let stopPressUp = triggerPressUp(e, 'virtual');
-            let stopPressEnd = triggerPressEnd(e, 'virtual');
+            let stopPressStart = triggerPressStart(createEvent(state.target, initEventCoordinates(e, state.target)), 'virtual');
```

Comment:
looks like this has introduced a bug, that's why there are some failing tests, you don't appear to be passing through the event.currentTarget anymore
so in triggerPressStart, it throws an error when the new PressEvent is created and we try to read the getBoundingClientRect

Reply:
If I understood you right it should be fixed now please have a look too.